metricbeat-tests/metricbeat.go

```diff
@@ -53,7 +46,19 @@ func RunMetricbeatService(version string, monitoredService services.Service) (se
 	log.WithFields(log.Fields{
 		"metricbeatVersion": version,
 		"service":           serviceName,
+		"serviceVersion":    monitoredService.GetVersion(),
 	}).Info("Metricbeat is running configured for the service")
 
+	log.WithFields(log.Fields{
+		"metricbeatVersion": version,
+		"service":           serviceName,
+	}).Debug("Installing Kibana dashboards")
+	docker.ExecCommandIntoContainer(
+		service.GetContainerName(), "root", []string{"metricbeat", "setup"})
```

Comment:
To avoid requiring Kibana for these tests, setup only indexes, that I think is all that is needed by now.

```suggestion
		service.GetContainerName(), "root", []string{"metricbeat", "setup", "--index-management"})
```

Reply:
Applied